openmp-17.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: openmp-17
+  version: 17.0.5
+  epoch: 0
+  description: "LLVM OpenMP library"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - cmake
+      - samurai
+      - binutils-dev
+      - libffi-dev
+      - zlib-dev
+      - libxml2-dev
+      - pkgconf
+      - python3
+      - help2man
+      - llvm17
```

Comment:
Do we need both llvm17 and llvm17-dev?

Reply:
yes we need both 

llvm17 provides `usr/lib/llvm17/bin/llvm-tblgen` which is needed here